x-pack/plugins/security_solution/public/timelines/store/timeline/reducer.ts

```diff
@@ -177,16 +178,18 @@ export const timelineReducer = reducerWithInitialState(initialTimelineState)
     ...state,
     timelineById: addTimelineNoteToEvent({ id, noteId, eventId, timelineById: state.timelineById }),
   }))
-  .case(toggleExpandedEvent, (state, { timelineId, event }) => ({
-    ...state,
-    timelineById: {
-      ...state.timelineById,
-      [timelineId]: {
-        ...state.timelineById[timelineId],
-        expandedEvent: event,
+  .case(toggleExpandedEvent, (state, { timelineId, event, defaultModel }) => {
+    return {
+      ...state,
+      timelineById: {
+        ...state.timelineById,
+        [timelineId]: {
+          ...(state.timelineById[timelineId] ?? defaultModel ?? timelineDefaults),
```

Comment:
the timeline should be intialized before calling `toggleExpandedEvent`, there should be no way to have an event expanded if the timeline doesn't exist.
It's also dangerous to setup the store with some side action